src/lib/y2partitioner/widgets/disk_expert_menu_button.rb

```diff
@@ -43,30 +48,41 @@ def label
       #
       # @return [Array<[Symbol, String]>]
       def items
-        [[:create_partition_table, _("Create New Partition &Table...")]]
+        [
+          [:create_partition_table, _("Create New Partition &Table")],
```

Comment:
why you remove that "..."?

Reply:
The string is in the old partitioner without dots.

Reply:
OK. JFYI in past we had convention that button that leads to another wizard had this "...", but we weren't much consistent with it.